### ISSUE TITLE
make cat/items tabbbabble 

### ISSUE THREAD

**Issue by ddwag1** [calify-io/calify#74]

**Comment by maxletourneur** [comment]
What does it mean ?

**Comment by ddwag1** [comment]
accessibility. When I press the tab button I should be able to cycle through categories on the categories page, and cycle through items on the category page
